### DISCUSSION TITLE
Turn off PV

### DISCUSSION THREAD

**Discussion by hemiceni** [mletenay/home-assistant-goodwe-inverter#253] [Q&A]
Should it be possible with this intergration to turn off/ disable PV arrays?

I frequently have a situation during the day where grid prices go negative but i cannot take advantage of it unless i turn off my PV otherwise they will service all my load.

If it cant be done in the intergration,  id love to hear ideas on how i can automate this some other way.

I also found this document online (see picture)
Do these entries refer to switching pv on and off via modbus?

![Screenshot_20240213_151838_Samsung Notes](https://github.com/mletenay/home-assistant-goodwe-inverter/assets/4637419/e97cb278-889d-468a-8955-908d1a6c3c9c)

**Comment by tinuva** [comment]
I don't know about that setting, but what is available today for me on the 5048 ES, is the "export limit" setting. See screenshot. You can just set this to 0 W. It will not disable PV, but it will block exporting to your utlity.

![Screenshot 2024-02-13 at 07 55 23](https://github.com/mletenay/home-assistant-goodwe-inverter/assets/5417682/b2350bdf-485f-4b3e-9ff8-43ef87bcd870)

**Comment by hemiceni** [comment]
I'm aware of export limit. Unfortunately that doesnt stop my PV from powering my whole house when what i actually want is to be 100% grid powered.

**Comment by lampra1** [comment]
[Load control](https://community.goodwe.com/solution/GoodWe%20Load%20Control%20Solution) seems to me more relevant than switching off the pv if you have an ET inverter. The document probably describes the state of the pv string rather than means to control it. To my knowledge, I have not seen any commercial inverter with remote on/off on the dc side. Alternatively, if your inverter has remote on/off on the ac side, maybe you could use that function.
